### PR TITLE
feat(player-manager): add OSD timeout to full screen video player

### PR DESCRIPTION
**Pull request by MrTimscampi**

### CODE REVIEW

**Review by ThibaultNocchi — APPROVED** [review]

**Comment by ThibaultNocchi** [comment]
Didn't test though, but it seems good to me

File: components/Players/PlayerManager.vue [hunk 1]
```diff
@@ -316,6 +335,11 @@ export default Vue.extend({
       }
     });
   },
+  beforeDestroy() {
```
Comment (ferferga) [inline components/Players/PlayerManager.vue:338]: We should remove the event handler here

**Review by ferferga — CHANGES_REQUESTED** [review]

File: components/Players/PlayerManager.vue (hunk 1 above)
Comment (ThibaultNocchi) [reply components/Players/PlayerManager.vue:323]: Isn't it already in L327?

Comment (MrTimscampi) [reply components/Players/PlayerManager.vue:323]: I updated the branch a bit after @ferferga mentioned it ;)

**Review by ThibaultNocchi — APPROVED** [review]

**Review by ferferga — APPROVED** [review]

**Pull request merged**